doc/design/vic-machine.md

```diff
@@ -179,6 +179,47 @@ Some of the elevated privilege operations could be delegated during self-provisi
 
 At this point install transitions to managing - reporting VCH status from initial install is the same as reporting that information for any VCH regardless of age.
 
+
+## Deleting - per vSphere target
+
+### Inputs
+
+1. vSphere SDK endpoint
+2. vSphere administrative credentials
+3. VCH identification
+ - VCH resource pool path in govc format, e.g. /ha-datacenter/vm/vch1
```

Comment:
the moref is a shorter form but doesn't really matter so long as it matches the output in `vic-machine ls` so copy-paste can be used.